lib/puppet/transaction/persistence.rb

```diff
@@ -64,7 +64,7 @@ def load
       begin
         result = Puppet::Util::Yaml.safe_load_file(filename, [Symbol])
       rescue Puppet::Util::Yaml::YamlLoadError => detail
-        Puppet.log_exception(detail, _("Transaction store file %{filename} is corrupt (%{detail}); replacing") % { filename: filename, detail: detail }, { :level => :warning })
+        Puppet.log_exception(detail, _("Transaction store file %{filename} is corrupt (%{detail}); replacing") % { filename: filename, detail: detail })
```

Comment:
The test for this specified logging at the error level, and the implementation tried to log as a warning, but that went through as an error due to the underlying problem I'm fixing here. I wasn't sure which to go with so I erred on the side of preserving existing behavior.